components/gpio_control/GPIODevices/simple_button.py

```diff
@@ -184,4 +184,6 @@ def __repr__(self):
     pin = int(input())
     func = lambda *args: print('FunctionCall with {}'.format(args))
     btn = SimpleButton(pin=pin, action=func, hold_mode='Repeat')
+
+    print('running')
```

Comment:
Should this be a log entry?

Reply:
The main method is just for testing so this is fine. But i would rather comment it out, so the test coverage can be achieved.
Its for manuell testing only anyway.